docs/custom_tasks.rst

```diff
@@ -8,11 +8,11 @@ Here is a simple example:
 .. literalinclude:: ../examples/tasks/timezone.py
 
 You must be sure that your code is compatible with the python binaries you use
-locally and remotly (2.x vs 3.x).
+locally and remotely (2.x vs 3.x).
 
 nuka's builtin tasks support python 2.7 and 3.4+
 
-Good practice wnat that your task are isolated in a ``tasks`` package and that
+Good practice want that your task are isolated in a ``tasks`` package and that
 they only use python's stdlib.
```

Comment:
As I'm creating a PR, could I suggested the following sentence instead ?

> As a good practice your task should be isolated in a ``tasks`` package and must only use python's stdlib.

Reply:
Sure, please & thank you